src/simpla-text-editor.html

```diff
@@ -2,6 +2,7 @@
 <link rel="import" href="../simpla-styles/simpla-styles.html">
 <link rel="import" href="../iron-iconset-svg/iron-iconset-svg.html">
 <link rel="import" href="../iron-icon/iron-icon.html" async>
+<link rel="import" href="../simpla-richtext-behavior/simpla-richtext-behavior.html">
```

Comment:
Thinking about this more it probably would be cleaner to now import this _alongside_ the editor component, and call this component `simpla-text-toolbar.html`. But same thing at the end of the day (well, not exactly the same, because this will delay the fetching of richtext until editor imported). 

Should also import richtext async and wait for it with pWaitFor or similar.